doc/modules/mod_pubsub.md

```diff
@@ -13,25 +13,109 @@ It's all about tailoring PubSub to your needs!
 
 ### Options
 
-* `iqdisc` (default: `one_queue`)
-* `host` (string, default: `"pubsub.@HOST@"`): Subdomain for Pubsub service to reside under.
+#### `modules.mod_pubsub.iqdisc.type`
+* **Syntax:** string, one of `"one_queue"`, `"no_queue"`, `"queues"`, `"parallel"`
+* **Default:** `"no_queue"`
+
+Strategy to handle incoming stanzas. For details, please refer to
+[IQ processing policies](../../advanced-configuration/Modules/#iq-processing-policies).
+
+#### `modules.mod_pubsub.host`
+* **Syntax:** string
+* **Default:** `"pubsub.@HOST@"`
+* **Example:** `host = "pubsub.localhost"`
+
+Subdomain for Pubsub service to reside under.
 `@HOST@` is replaced with each served domain.
-* `backend` (atom, default: `mnesia`) - Database backend to use. `mnesia` and `rdbms` are supported currently.
-* `access_createnode` (atom, default: `all`): Who is allowed to create pubsub nodes.
-* `max_items_node` (integer, default: `10`): Define the maximum number of items that can be stored in a node.
-* `max_subscriptions_node` (integer, default: `undefined` - no limitation): The maximum number of subscriptions managed by a node.
-* `nodetree` (binary, default: `<<"tree">>`): Specifies the storage and organisation of the pubsub nodes. See the section below.
-* `ignore_pep_from_offline` (boolean, default: `true`): specify whether or not we should get last published PEP items from users in our roster which are offline when we connect.
+
+#### `modules.mod_pubsub.backend`
+* **Syntax:** string, one of `"mnesia"`, `"rdbms"`
+* **Default:** `"mnesia"`
+* **Example:** `backend = "rdbms"`
+
+Database backend to use.
+
+#### `modules.mod_pubsub.access_createnode`
+* **Syntax:** string
+* **Default:** `"all"`
+* **Example:** `access_createnode = "all"`
+
+Specifies who is allowed to create pubsub nodes.
+
+#### `modules.mod_pubsub.max_items_node`
+* **Syntax:** non-negative integer
+* **Default:** `10`
+* **Example:** `max_items_node = 10`
+
+Defines the maximum number of items that can be stored in a node.
+
+#### `modules.mod_pubsub.max_subscriptions_node`
+* **Syntax:** non-negative integer or the string `"undefined"`
```

Comment:
"undefined"` wouldn't pass the validation, right?

Reply:
```suggestion
* **Syntax:** non-negative integer
```